src/components/biblio/topic_entity_tag/TopicEntityTable.js

```diff
@@ -23,9 +15,9 @@ import "ag-grid-community/styles/ag-theme-quartz.css"; // Theme
 const TopicEntityTable = () => {
   const dispatch = useDispatch();
   const accessToken = useSelector(state => state.isLogged.accessToken);
-  const oktaMod = useSelector(state => state.isLogged.oktaMod);
-  const testerMod = useSelector(state => state.isLogged.testerMod);
-  const accessLevel = (testerMod !== 'No') ? testerMod : oktaMod;  
+  //const oktaMod = useSelector(state => state.isLogged.oktaMod);
+  //const testerMod = useSelector(state => state.isLogged.testerMod);
+  //const accessLevel = (testerMod !== 'No') ? testerMod : oktaMod;
```

Comment:
I guess we don't need okta/access because the delete functionality is being removed ?  I've lost track of what it should do, so if curators no longer want that, ok.

Reply:
The delete function is now in another file... so we dont need it here.  All that code is in TopicEntityTagActions.jsx.  That is the cell renderer that builds the values in that cell.

Reply:
cool, thanks